### PR TITLE
Adds test cases for edged integration test

### LINKED ISSUE #151: Integration test enhancement for DeviceTwin and edged
**What would you like to be added**:

More integration tests required for DeviceTwin and edged

**Why is this needed**:

DeviceTwin and Edged have various functionalities, and changes in one file may affect other functionality which depends on that file. Currently we do not have tests for many features/functionality of deviceTwin and edged. Only basic tests are present which do not test all the scenarios.

### PR DESCRIPTION
**Pull request by maurya-anuj**
New test cases for integration test of edged
Fixes #151

**What type of PR is this?**
>  test

**What this PR does / why we need it**: Adds new test cases for edged integration test
Fixes #151

### CODE REVIEW

File: edge/test/integration/appdeployment/application_test.go [hunk 1]
```diff
@@ -24,8 +24,12 @@ import (
 	"github.com/kubeedge/kubeedge/edge/test/integration/utils/edge"
 	. "github.com/kubeedge/kubeedge/edge/test/integration/utils/helpers"
 
+	"k8s.io/apimachinery/pkg/util/intstr"
+
```
Comment (sids-b) [inline edge/test/integration/appdeployment/application_test.go]: Delete blank line

File: edge/test/integration/utils/helpers/helpers.go [hunk 2]
```diff
@@ -261,4 +261,4 @@ func CheckPodDeletion(EdgedEndPoint, UID string) {
 		}
 		return IsExist
 	}, "30s", "4s").Should(Equal(false), "Delete Application deployment is Unsuccessfull, Pod has not come to Running State")
-}
+}
```
Comment (m1093782566) [inline edge/test/integration/utils/helpers/helpers.go]: nit: a newline please.

**Pull request merged**